internal/connector/pingone/resources/platform/pingone_agreement_localization_revision.go

```diff
@@ -1,9 +1,10 @@
-package pingoneplatformresources
+package platform
 
 import (
 	"fmt"
 
 	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/pingone/resources/common"
 	pingoneresources "github.com/pingidentity/pingctl/internal/connector/pingone/resources/common"
```

Comment:
Two imports for the same thing here (and possibly in other resources)

Reply:
this should be resolved